src/main/java/ay2122s1_cs2103t_w16_2/btbb/model/predicate/PredicateCollection.java

```diff
@@ -18,18 +18,18 @@
  * @param <T> Type of the predicate.
```

Comment:
What is the reason for this change? Can you revert back? We can add it later in v1.3 if you want.

Reply:
I believe this is so that when compiling it does not show the warning